docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     environment:
       SPATIAL_URL: http://spatial
       GEOSERVER_ROOT: data/geoserver/
-      GEOSERVER_SECURE_URL: http://admin:geoserver@geoserver_secure:8080/geoserver/rest/
+      GEOSERVER_SECURE_URL: http://admin:geoserver@geoserver-secure:8080/geoserver/rest/
```

Comment:
It was necessary to change this because it's not actually a valid URI (hyphens are allowed, but not underscores), which caused Ruby's URI module to raise errors.